frontend/public/components/_detail-table.scss

```diff
@@ -53,3 +53,10 @@
     text-align: left;
   }
 }
+
+.image-stream-manipulation__tooltip {
+  width: 800px;
```

Comment:
We should find some styles that work on mobile, too. @rhamilto or @sg00dwin might be able to help.

Reply:
PF4 has a variants with an expansion if the text is too long, but I'm not sure if it's smart enough to switch from the simple to the expandable if the text wraps. Seems like that'd be ideal.

http://patternfly-react.surge.sh/patternfly-4/components/clipboardcopy/

Reply:
Wondering if a modal wouldn't be better here given the amount of content?

Reply:
I guess my main argument against a modal is that in this case we don't need to block users, or force them into doing anything/make a choice. It's just optional information, so I like that users can easily dismiss the popover. The can: click outside of the popover, click the x, or click the link again (toggle) to get rid of it. AFAICT from the demos, the PF4 modal forces a user to either click the x or click a button to dismiss, so there's a little more friction there which I'm not sure is necessary here.

But also, we're thinking of this as a pattern for providing in-context help that's available when/where needed. So I'm not sure where the line would be to use modal v popover. I'm still working on clarifying what the pattern should be, so I'm not 100% opposed to a modal.